kroxylicious-kms-provider-aws-kms/src/main/java/io/kroxylicious/kms/provider/aws/kms/model/DescribeKeyResponse.java

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright Kroxylicious Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+package io.kroxylicious.kms.provider.aws.kms.model;
+
+import java.util.Objects;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+public record DescribeKeyResponse(@JsonProperty(value = "KeyMetadata") KeyMetadata keyMetadata) {
```

Comment:
I'll make this point here once, but I guess it applies to may of your other JSON serializable records

`@JsonProperty(value = "KeyMetadata", required=true) @NonNull`

Reply:
Done